dfp.module

```diff
@@ -47,6 +47,8 @@ function dfp_theme() {
         'single_request' => NULL,
         'collapse_empty_divs' => NULL,
         'disable_init_load' => NULL,
+        'slug_position' => NULL,
```

Comment:
Passing these variables to the template will help to decide when to add the scripts